React/Modules/RCTUIManager.m

```diff
@@ -1143,6 +1143,11 @@ - (void)batchDidComplete
   [self _layoutAndMount];
 }
 
+RCT_EXPORT_METHOD(playTouchSound)
```

Comment:
I dislike an idea to put something related to sound into `RCTUIManager`. We just have to have dedicated AudioManager (if we still don't have one) and then implement whole feature on Javascript side.